### ISSUE TITLE
Heavyweight Bayesian scheduling prediction

### ISSUE THREAD

**Issue by brownbat** [brownbat/autoEaseFactor#8]
Ebisu provides a model for more sophisticated forgetting prediction and scheduling:
https://fasiha.github.io/ebisu/

Incorporating those ideas, including information about reviews that came earlier or later than expected, would make this algorithm more powerful. Adding small random variations in the scheduling to help catch both type i and type ii errors more equally would also be helpful.

**Comment by S0rter** [comment]
Someone has tried this before: https://github.com/thetruejacob/Anki-Ebisu It doesn't work on the current anki version but maybe the people behind the project or at least their code is helpful to you.

**Comment by brownbat** [comment]
Yeah, good link, I had seen that, but more people should be aware of that version. 

This is still back burner for me mainly because one of the ebisu assumptions is that learning interval is fixed, and it is slow to update if you aren't reviewing a card that much after it's deadline.

It's an absolutely brilliant project, so I don't mean to detract from it at all, but before I integrate I'd have to make some major changes to fit the reviewing model and assumptions I'm working with here.

I still hope to cross this bridge someday, and thetruejacob's efforts will be a big help. (I really need to take some classes from Brilliant on multiple regression and Bayesian probability... so hard to learn a bunch of things with a full time job... )

More discussion: 
https://github.com/fasiha/ebisu/issues/35#issue-688083344
